baseapp-cloudflare-stream-field/baseapp_cloudflare_stream_field/stream.py

```diff
@@ -42,6 +42,14 @@ def update_video_data(self, video_uid, data={}):
         )
         return res.json()["result"]
 
+    def upload_caption_file(self, video_uid, language_code, caption_file):
+        res = requests.put(
+            f"{self._api_url}/{video_uid}/captions/{language_code}",
+            headers={key: value for key, value in self.request_headers.items() if key != "Content-Type"},
```

Comment:
Maybe instead of looping through `self.request_headers` you could instead do:

```python
headers={
    **self.request_headers,
    "Content-Type": "multipart/form-data"
}
```

Reply:
Good idea. check now